src/main/java/com/google/sps/data/InsertAndUpdate.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+public class InsertAndUpdate{
+
+    private static final String REP_ENTITY_TYPE = "Representative";
+    private static final String REP_NAME = "Name";
+    private static final String REP_TITLE = "Official Title";
+    private static final String REP_POSTS = "Posts";
+
+    private static final String POST_ENTITY_TYPE = "Post";
+    private static final String POST_QUESTION = "Question";
+    private static final String POST_ANSWER = "Answer";
+    private static final String POST_REPLIES = "Replies";
+
+    private static final String COMMENT_ENTITY_TYPE = "Comment";
+    private static final String COMMENT_NAME = "Nick Name";
+    private static final String COMMENT_MSG = "Message";
```

Comment:
I moved all the constants to a separate class in my PR. How should we avoid the tons and tons of merge conflicts we are bound to have?  I also had to change some of the functions when testing and change variables to camel-case and stuff. I can send you the file or you can remove from PR and it'll be fine when we merge

Reply:
I honestly don't think the merge will be too bad - we'll just keep your version but either way, you can send file and I'll update if we have time before we integrate